planner/cascades/integration_test.go

```diff
@@ -292,3 +292,27 @@ func (s *testIntegrationSuite) TestTopN(c *C) {
 		tk.MustQuery(sql).Check(testkit.Rows(output[i].Result...))
 	}
 }
+
+func (s *testIntegrationSuite) TestInlineProjection(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	tk.MustExec("drop table if exists t1, t2;")
+	tk.MustExec("create table t1(a bigint, b bigint, index idx_a(a), index idx_b(b));")
+	tk.MustExec("create table t2(a bigint, b bigint, index idx_a(a), index idx_b(b));")
```

Comment:
Would you please add some data to the tables and make results not null?

Reply:
done